watchfaces/008-funky-town-words.qml

```diff
@@ -62,19 +62,23 @@ Item {
     }
 
     Text {
-        z: 1
         id: minuteDisplay
-        font.pixelSize: parent.height*0.24
+
+        property bool animToggle: true
+        onTextChanged: if (!displayAmbient) { animToggle ? animToggle = false : animToggle = true }
+
+        z: 1
+        font.pixelSize: parent.height * 0.22
         font.family: "Source Sans Pro"
         font.styleName: "Light"
         color: "white"
         anchors {
             bottom: parent.bottom
-            bottomMargin: parent.height*0.155
+            bottomMargin: parent.height * 0.15
             horizontalCenter: parent.horizontalCenter
-            horizontalCenterOffset: parent.width*0.210
+            horizontalCenterOffset: parent.width*0.2
         }
-        Behavior on text {
+        Behavior on animToggle {
```

Comment:
I think I understand the `displayAmbient` issue: The watchface is trying to animate while the display isn't active hence resulting in a `minuteDisplay` that has a low opacity.

There might be a simpler solution to this though: Why not disable the `Behavior` animation when in `displayAmbient` mode. i.e add a line below: `enabled: !displayAmbient`.

Reply:
I tried `running: !displayAmbient`... And then came up withthe workaround when it did not work...
Thanks for the hint.